email_handler.py

```diff
@@ -2078,6 +2079,20 @@ def should_ignore(mail_from: str, rcpt_tos: List[str]) -> bool:
     return False
 
 
+def send_no_reply_response(mail_from: str, msg: Message):
+    user = User.get_by(email=mail_from)
```

Comment:
It's possible that user replies to a noreply address from a mailbox which isn't his SL account address. I think we can maybe create a similar method to `send_email_at_most_times` which doesn't require the `user` parameter. This methods use `SentAlert` for rate limiting and SentAlert.user_id attribute is optional.

Reply:
How about trying to find out if there's a user with that email registered. And send it only once per user still. If we don't know the mail for any of our users it's probably safe to just skip it.